apis/pkg/v1/conditions.go

```diff
@@ -104,3 +140,50 @@ func UnknownHealth() xpv1.Condition {
 		Reason:             ReasonUnknownHealth,
 	}
 }
+
+// VerificationSucceeded returns a condition indicating that a package's
+// signature has been successfully verified using the supplied image config.
+func VerificationSucceeded(imageConfig string) xpv1.Condition {
+	return xpv1.Condition{
+		Type:               TypeSignatureVerificationComplete,
+		Status:             corev1.ConditionTrue,
+		LastTransitionTime: metav1.Now(),
+		Reason:             ReasonVerificationSucceeded,
+		Message:            fmt.Sprintf("Signature verification succeeded with ImageConfig named %q", imageConfig),
+	}
+}
+
+// VerificationFailed returns a condition indicating that a package's
+// signature verification failed using the supplied image config.
+func VerificationFailed(imageConfig string, errors []error) xpv1.Condition {
```

Comment:
Initially, I designed to get multiple errors since there could be mutiple verification errors out of multiple authority checks but now seeing I am already joining those errors early in process and always pass a single error to this condition. So, changed this to get a single error.